### PR TITLE
Preparing for V1.4.0

### PR DESCRIPTION
**Pull request by falaki**
Updating build and README files.

### CODE REVIEW

**Comment by falaki** [comment]
@HyukjinKwon Would you like to review this?

**Comment by HyukjinKwon** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ version: 1.3.0
 ```
 groupId: com.databricks
 artifactId: spark-csv_2.11
-version: 1.3.0
+version: 1.4.0
```
Comment (HyukjinKwon) [inline README.md]: You forgot to correct the one for `spark-csv_2.10`.

**Comment by HyukjinKwon** [comment]
Overall, LGTM.

File: README.md (hunk 1 above)
Comment (falaki) [inline README.md]: Thanks!

**Pull request closed without merging**